SampleApps/SampleApps/SampleApp1/main.cpp

```diff
@@ -19,8 +28,247 @@
 
 #include <VbsEnclave\HostApp\Stubs\Trusted.h>
 
+using namespace winrt::Windows::Security::Credentials;
+
 namespace fs = std::filesystem;
 
+// Global variables (declared but not initialized, need to be encapsulated)
+std::wstring g_helloKeyName;
+std::wstring g_pinMessage;
+std::optional<winrt::Windows::Security::Credentials::KeyCredentialCacheConfiguration> g_keyCredentialCacheConfig;
+HWND g_hCurWnd;
+
+// Initialize function to set up global variables
+void Initialize()
```

Comment:
nit: `Initialize` feels too formal and disjoint from the sample logic, imo.  Can problably remove this.